server/assessmentsDb.js

```diff
@@ -0,0 +1,19 @@
+const environment = process.env.NODE_ENV || 'development'
+const config = require('./knexfile')[environment]
+const connection = require('knex')(config)
+
+function getAssessments(id, db = connection) {}
+
+function updateEvidence(assId, evidenceInfo, db) {
```

Comment:
I would prefer the following parameter names:
```
(id, evidence, db = connection)
```

Reply:
updated variable names